stage3-py.sh

```diff
@@ -7,14 +7,12 @@ set -e
 #
 # source ${LOADSTACK}
 # rspname="rsp-${LSST_CONDA_ENV_NAME}"
-# conda create --name ${rspname} --clone ${LSST_CONDA_ENV_NAME}
+# mamba create --name ${rspname} --clone ${LSST_CONDA_ENV_NAME}
 #
 source ${LOADRSPSTACK}
-conda install -y mamba # not strictly necessary, but better error reporting
 mamba install --no-banner -y \
       "jupyterlab>=3,<4" \
       ipykernel \
-      jupyterhub \
```

Comment:
I take it the hub is only needed in the service container, not the invoked containers.